src/python_testing/TC_IDM_1_4.py

```diff
@@ -25,12 +25,10 @@
 from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main, type_matches
```

Comment:
Adding blocking comment so that we don't merge until https://github.com/CHIP-Specifications/chip-test-plans/pull/3937 is merged